clientd3d/d3drender.c

```diff
@@ -3316,6 +3316,7 @@ void D3DRenderNamesDraw3D(d3d_render_cache_system *pCacheSystem, d3d_render_pool
 	for (list = room->contents; list != NULL; list = list->next)
 	{
       float glyph_scale = 255;
+      bool bUsingAlternateDepth;

```

Comment:
Add a comment: what is alternate depth?

Also, move declaration of variable to its first use.